packages/xod-arduino-deploy/src/messages.js

```diff
@@ -0,0 +1,15 @@
+export const consoleMessages = {
+  // Check does library installed
+  libraryAlreadyInstalled: libName => `Library "${libName}" already installed`,
+  libraryAreNotInstalled: libName => `Library "${libName}" are not installed`,
+  libraryDownloaded: libName => `Library "${libName}" downloaded`,
+  libraryInstalled: libName => `Library "${libName}" installed successfully`,
+};
+
+export default {
+  // Get library name from url
+  CANT_GET_LIBRARY_NAME: ({ url }) => ({
+    title: 'Wrong library URL or library name',
+    note: `Tried to download library from "${url}"`,
+  }),
+};
```

Comment:
What the new pattern? Should it be just a plain list of functions to format this and that? The last time a `messages.js` mixed several concepts it was quite hard to repair. Remember that `[object Object]` errors?!

Reply:
🤔 
This is the same one, that we used last time in the `xod-project`.

Reply:
Oops, it was possibly overlooked